CMakeLists.txt

```diff
@@ -13,21 +13,40 @@ if(NOT COMMAND add_llvm_library)
         # We need to match the build environment for LLVM:
         # In particular, we need C++11 and the -fno-rtti flag
         set(CMAKE_CXX_STANDARD 14)
-	if(CMAKE_BUILD_TYPE MATCHES "Debug")
-		set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++14 -g -O0 -fno-rtti -Wno-deprecated -DHAVE_IEEE_754 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8")
-	else()
-		set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++14 -O3 -fno-rtti -Wno-deprecated -DHAVE_IEEE_754 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8")
-	endif()
-	set(CMAKE_C_FLAGS "-fPIC")
```

Comment:
Can we still keep -fPIC or did I miss anything in your patch?

Reply:
PIC is enabled by default on recent Linux distributions, to support ASLR.
So it is duplicated.

Reply:
Just in case, someone is built on an old version of Linux.

Reply:
Yes. Update.
Please have a look.
Also, I removed `-Wno-deprecated`, as the current code can be built now without it.